tools/check_format_test_helper.py

```diff
@@ -79,19 +82,19 @@ def fixFileExpectingNoChange(file):
     return 1
   return 0
 
-def emitStdout(stdout):
+def emitStdoutAsError(stdout):
   for line in stdout:
-    print("    %s" % line)
+    logging.error("    %s" % line)
```

Comment:
nit: i'd rather this return a newline joined string and then call `logging.error(stdoutAsString(stdout))` inline